### PR TITLE
switch from port 3000 to 9926

### PR DESCRIPTION
**Pull request by Ethan-Arrowood**

### CODE REVIEW

File: extension.js [hunk 1]
```diff
@@ -75,7 +75,7 @@ function resolveConfig(options) {
 		buildOnly: options.buildOnly ?? false,
 		dev: options.dev ?? false,
 		installCommand: options.installCommand ?? 'npm install',
-		port: options.port ?? 3000,
+		port: options.port ?? 9926,
```
[the remark this answers is not in the document]
Comment (kriszyp) [reply extension.js:78]: Also, while we are at it, we should probably add `securePort: options.securePort,`

[the remark this answers is not in the document]
Comment (Ethan-Arrowood) [reply extension.js:78]: And so port is not optional? It must be configured? Or do you mean for it to inherit this option from somewhere else?

[the remark this answers is not in the document]
Comment (kriszyp) [reply extension.js:78]: It is optional, and if it is omitted (or undefined-ish), then it is inherited from the main http.port/http.securePort configuration.

[the remark this answers is not in the document]
Comment (Ethan-Arrowood) [reply extension.js:78]: Oh I see. I didn't realize the options were passed through like that. I'll make the change now

**Review by kriszyp — APPROVED** [review]

**Comment by Ethan-Arrowood** [comment]
This isn't working yet; something is causing it to still run on 3000.

**Pull request merged**